<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICcard</class>
 <widget class="QDialog" name="ICcard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="rechargeButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string> 充值</string>
   </property>
  </widget>
  <widget class="QPushButton" name="queryButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_show">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_tip">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
